eval/testdata/eval/schema-error/expected.json

```diff
@@ -21,7 +21,7 @@
             "Expression": null,
             "EvalContext": null,
             "Extra": null,
-            "Shown": false
+            "Path": "values.sink.fn::open::schema.boolean"
```

Comment:
will these paths parse as property paths? feels like these should probably be `values.sink["fn::open::schema"].boolean`

Reply:
good catch